### ISSUE TITLE
UI rendered incorrectly on mobile

### ISSUE THREAD

**Issue by Forceu** [Ravinou/borgwarehouse#244]
On mobile phones the UI is not usable, as only the red/green status and the repo names are shown. No other details are available and the side bar is hidden. This can be reproduced on Firefox or Chrome by resizing the window

![image](https://github.com/Ravinou/borgwarehouse/assets/1593467/f223c145-d4d6-4bf6-b101-4fe9382cd107)

**Comment by Ravinou** [comment]
Hi @Forceu, it's limited by design. [Check the documentation here.](https://borgwarehouse.com/docs/user-manual/mobile/)

**Comment by Forceu** [comment]
Thank you for the clarification. Personally I find it quite important - would you be willing to merge a PR at some point or do you want to keep it that way?

**Comment by Ravinou** [comment]
I've got a lot of work to do (and not much time) before tackling such a priority. In particular, I'd like to migrate to Typescript (I've started that) and integrate tests. That'll give me more peace of mind when it comes to UI changes like mobile too. So it's something that will be done.

By the way, thanks a lot for the idea of the environment variable to prevent deletion of repo. I hope to have a little more time this weekend to integrate this and merge your PR.
